### PR TITLE
Display client secret after creation per environment.

### PR DESCRIPTION
**Pull request by glucaci**

### CODE REVIEW

**Review by PascalSenn — DISMISSED** [review]

File: src/UI/src/components/Application/EditApplicationView.vue [hunk 1]
```diff
@@ -383,11 +383,14 @@ export default {
...
     async onAddEnvironment(environmentId) {
-      const application = await this.addEnvironmentToApplication({
+      await this.addEnvironmentToApplication({
         id: this.id,
         environments: [environmentId],
       });
-      this.setApplication(application);
+
+      this.$router.replace({
+        name: "Application_Created",
```
Comment (PascalSenn) [inline src/UI/src/components/Application/EditApplicationView.vue:392]: was genau macht das?

File: src/UI/src/store/applicationStore.js [hunk 2]
```diff
@@ -45,6 +45,13 @@ const applicationStore = {
       state.list.items.push(result.application);
       state.lastCreated = result;
     },
+    APPLICATION_ENVIRONMENT_CREATED(state, result) {
+      const index = state.list.items.findIndex(x => x.id === result.application.id);
+      if (index > -1) {
+        Vue.set(state.list.items, index, result.application);
+      }
```
Comment (PascalSenn) [inline src/UI/src/store/applicationStore.js:52]:
```suggestion
      this.APPLICATION_UPDATED(state, result.application)
```

**Review by PascalSenn — COMMENTED** [review]
would this be better?

File: src/UI/src/components/Application/EditApplicationView.vue (hunk 1 above)
Comment (glucaci) [reply src/UI/src/components/Application/EditApplicationView.vue:392]: navigate to the page where the new created client is shown

File: src/UI/src/store/applicationStore.js (hunk 2 above)
Comment (glucaci) [reply src/UI/src/store/applicationStore.js:52]: nice, thanks

**Review by PascalSenn — APPROVED** [review]

**Pull request merged**